.github/workflows/tarball.yml

```diff
@@ -0,0 +1,247 @@
+name: CmdStan tarball builder
+
+'on':
+  workflow_dispatch:
+    inputs:
+      tarball_name: 
+        description: 'CmdStan build name'
+        required: true
+        default: ''
+      cmdstan_repo:
+        description: 'CmdStan repo'
+        required: true
+        default: 'stan-dev/cmdstan'
+      cmdstan_branch:
+          description: 'CmdStan branch'
+          required: true
+          default: 'develop'
+      stan_repo:
+        description: 'Stan repo'
+        required: true
+        default: 'stan-dev/stan'
+      stan_branch:
+          description: 'Stan branch'
+          required: true
+          default: 'develop'
+      math_repo:
+        description: 'Math repo'
+        required: true
+        default: 'stan-dev/math'
+      math_branch:
+          description: 'Math branch'
+          required: true
+          default: 'develop'
+      stanc3_repo:
+        description: 'Stanc3 repo'
+        required: true
+        default: 'stan-dev/stanc3'
+      stanc3_branch:
+          description: 'Stanc3 branch'
+          required: true
+          default: 'master'
+jobs:
+  linux:
+    runs-on: ubuntu-latest
```

Comment:
What if we start to enforce a policy to specify the linux build instead of relying in `ubuntu-latest`? Maybe `ubuntu-20.04`? GitHub updates which distribution is `ubuntu-latest` every now and then (in October 2020 they announced that `ubuntu-latest` would become `ubuntu-20.04`; see [here](https://github.blog/changelog/2020-10-29-github-actions-ubuntu-latest-workflows-will-use-ubuntu-20-04/)).